adhocracy-plus/templates/header_lower.html

```diff
@@ -5,36 +5,34 @@
     <div class="col-md-10 offset-md-1">
         <nav class="header-lower__nav">
 
-            {% if ORGANISATION and ORGANISATION.logo %}
-              {% url 'organisation' organisation_slug=ORGANISATION.slug as organisation_url %}
-                <a href="{% url 'organisation' organisation_slug=ORGANISATION.slug %}" class="header-lower__logo-box" rel="home">
-                    <img src="{% thumbnail ORGANISATION.logo '0x160' %}" height="80" alt="{{ ORGANISATION.name }}" class="header-lower__logo-img" />
+            {% if organisation.logo %}
+              {% url 'organisation' organisation_slug=organisation.slug as organisation_url %}
+                <a href="{% url 'organisation' organisation_slug=organisation.slug %}" class="header-lower__logo-box" rel="home">
+                    <img src="{% thumbnail organisation.logo '0x160' %}" height="80" alt="{{ organisation.name }}" class="header-lower__logo-img" />
                 </a>
-            {% elif ORGANISATION %}
+            {% else %}
```

Comment:
I get an error if the logo is not there, because the slug is missing.. probably only because of the capital letters?

Reply:
Hmm, no, I just changed that back from the fix before (where the organisation just isn't shown, when it's wrong in the url). What exactly is the error?

Reply:
Oh, or maybe in some views it does need the correct organisation slug from the url? Which view was that in?

Reply:
The error is `Reverse for 'organisation' with keyword arguments '{'organisation_slug': ''}' not found. 1 pattern(s) tried: ['(?P<organisation_slug>[-\\w_]+)/$']` in the header_lower.html 

But sorry, I commented in the wrong line, it is line 15 I meant ..